### PR TITLE
[11.0][IMP] stock_batch_picking: Make move_line_ids writable

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Is this working correctly? I see several problems:

* A computed one2many field is doomed to fail in some processes, as `one2many` uses the inverse name for doing certain operations. You should switch to many2many.
* Switching to many2many, when editing, you already have it editable.
* Adding lines is not possible (should be limited), because which picking is the source? This can be bypassed adding the picking field, but it's clumsy.

**Comment by rousseldenis** [comment]
@sergio-teruel Can you attend @pedrobaeza comments ?

**Comment by pedrobaeza** [comment]
@sergio-teruel don't make translations on a PR, but on Weblate. This will conflict for sure, and more as the PR is not going to be merged soon, as the comments are not attended.

**Comment by rousseldenis** [comment]
@sergio-teruel Is it normal that you include several modifications that are not related to this PR? Moreover, they are mixed within same commits. I think about someone who should complain about that :smile:

File: stock_batch_picking/models/stock_batch_picking.py [hunk 1]
```diff
@@ -78,9 +78,9 @@ class StockBatchPicking(models.Model):
 
     move_line_ids = fields.One2many(
         'stock.move.line',
-        readonly=True,
         string='Related pack operations',
-        compute='_compute_move_line_ids'
+        compute='_compute_move_line_ids',
+        inverse=lambda self: self,
```
Comment (rousseldenis) [inline stock_batch_picking/models/stock_batch_picking.py:84]: @sergio-teruel Why this ?

**Comment by pedrobaeza** [comment]
@rousseldenis I have been reviewing with @sergio-teruel this PR, and first of all, the rest of unrelated modifications are now on separate PRs. Can you please review them?

About this one, the question is that you can't edit any of the `stock.move.line` fields. For example, "Done" quantity field, so with this little hack, we allow that. We are not able to find a better solution (tried with inverse over the fields theirselves, with regular inverse method, etc...).

What has been done now is that the computed fields are many2many, not one2many.

**Review by pedrobaeza — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request closed without merging**